pages/views/roles/data.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Data View
+permalink: views/roles/data
+---
+
+# Data View
+
+{% include carbonStandard.html variant=site.data.overlays.data hideOutOfScope=true %}
```

Comment:
Upstream Server and Storage Hardware and the SaaS link aren't working (linking to the page heading)
Cat O Networking and Infrastructure is linking to the default page (I think Matt spotted this already)